recipes/clonalframeml/1.11/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+export CFLAGS="-I$PREFIX/include"
```

Comment:
Are these exports needed? You do not depend on any third party library as it seems.